packages/db/prisma/schema.prisma

```diff
@@ -46,6 +46,25 @@ model Keyword {
     updatedAt DateTime @updatedAt
 }
 
+model Collections {
```

Comment:
can you make all table names singular for formality? So `Collection` lng then `TestsOnCollections`